cli/lib/command/internal_command_pr_creation.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:io';
+
+import 'package:stax/command/internal_command.dart';
+import 'package:stax/context/context.dart';
+import 'package:stax/context/context_git_get_current_branch.dart';
+import 'package:stax/context/context_git_get_default_branch.dart';
+import 'package:stax/context/context_git_is_inside_work_tree.dart';
+
+class InternalCommandPrCreation extends InternalCommand {
+  InternalCommandPrCreation()
+      : super(
+          'pr',
+          'Creates a pull request.',
+        );
+
+  @override
+  void run(final List<String> args, final Context context) {
+    if (context.handleNotInsideGitWorkingTree()) {
+      return;
+    }
+
+    final currentBranch = context.getCurrentBranch();
+    if (currentBranch == null) {
+      context.printToConsole("Can't determine current branch.");
+      return;
+    }
+
+    final defaultBranch = context.getDefaultBranch();
```

Comment:
We do not want PR to point at default brunch all the time. If this branch is stacked on another branch we want PR to point on stacked branch instead. 

In order to achieve this you can use `context.gitLogAll().collapse()` to find current and parent branch.